front/public/locales/fr/translation.json

```
@@ -50,6 +50,12 @@
         "title": "Echec de l'enregistrement des modifications",
         "text": "Vos modifications n'ont pas été publiées"
       }
+    },
+    "units": {
+      "day": "j",
+      "hour": "h",
+      "minute": "min",
+      "km": "km"
```

For alphabetical order :) (same for en)

```suggestion
      "km": "km",
      "minute": "min"
```